### PR TITLE
use body & title to parse jira ticket

### PR DESCRIPTION
**Pull request by jffmrk**
many of the iOS PRs do not have a Jira number in the body, but we do have one in the title of the PR.

### CODE REVIEW

File: bin/rngen [hunk 1]
```diff
@@ -215,7 +215,7 @@ for line in "${changelog_json[@]}"; do
   user="$(echo "$line" | jq -r .user.login)"
   body="$(echo "$line" | jq -r .body)"
   gh_url="$(echo "$line" | jq -r .html_url)"
-  jira_issue="$(echo "$body" | grep -oE '[A-Z]{2,8}-[0-9]{1,6}' | head -1)"
+  jira_issue="$(echo "$body $title" | grep -oE '[A-Z]{2,8}-[0-9]{1,6}' | head -1)"
```
Comment (justinhoward) [inline bin/rngen]: Want to put the title first? Perhaps if there's a ticket ref in the title, it should be prioritized.

**Review by justinhoward — COMMENTED** [review]
Awesome, thanks! One minor question, but looks great.

File: bin/rngen (hunk 1 above)
Comment (jffmrk) [reply bin/rngen]: sure 👍

**Review by justinhoward — APPROVED** [review]

**Comment by justinhoward** [comment]
🎉 Thanks!

**Pull request merged**